src/main/kotlin/com/avail/utility/dot/DotWriter.kt

```diff
@@ -99,12 +99,17 @@ import kotlin.math.min
  *   block comments.
  * @param accumulator
  *   The accumulator for the generated source code.
+ * @param darkMode
+ *   If true, draw with light colors on a dark background.
+ * @param copyrightOwner
+ *   This name is embedded in a copyright claim within the document.
  */
 class DotWriter constructor(
 	private val name: String,
 	internal val isDirected: Boolean,
 	internal val charactersPerLine: Int,
 	internal val accumulator: Appendable,
+	private val darkMode: Boolean = false,
```

Comment:
Might as well make dark mode the default. =)

Reply:
Done.